pylabrobot/liquid_handling/backends/tecan/EVO_backend.py

```diff
@@ -485,22 +545,63 @@ async def drop_tips(self, ops: List[Drop], use_channels: List[int]):
     assert (
       min(use_channels) >= self.num_channels - self.diti_count
     ), f"DiTis can only be configured for the last {self.diti_count} channels"
-    assert all(isinstance(op.resource, Trash) for op in ops), "Must drop in waste container"
+    assert all(
+      isinstance(op.resource, (Trash, TipSpot)) for op in ops
+    ), "Must drop in waste container or tip rack"
 
-    x_positions, y_positions, _ = self._liha_positions(ops, use_channels)
+    # Get positions including offsets
+    x_positions, y_positions, z_positions = self._liha_positions(ops, use_channels)
+
+    # Apply offsets
+    for i, op in enumerate(ops):
+      x_positions[i] += op.offset.x
+      y_positions[i] += op.offset.y
+
+    for key in z_positions:
+      z_positions[key][i] += op.offset.z  # Apply the offset to all z position types
 
     # move channels
-    ys = 90
+    ys = int(ops[0].resource.get_absolute_size_y() * 10)  # was 90
+    x, _ = self._first_valid(x_positions)
+    y, yi = self._first_valid(y_positions)
+    assert x is not None and y is not None
+    await self.liha.set_z_travel_height([self._z_range] * self.num_channels)
+    await self.liha.position_absolute_all_axis(
+      x,
+      y - yi * ys,
+      ys,
+      [self._z_range] * self.num_channels,
+    )
+
+    # TODO check channel positions match resource positions for z-axis
+    await self.liha.discard_disposable_tip(self._bin_use_channels(use_channels), discard_hight=0)
```

Comment:
binary:
- discard_hight = 0: into the box
- discard_hight = 1: above the box

@VilhelmKMoller will test

maybe in the future:

```
if resource is trash:
  above the box
else:
  in the box
```